perun-rpc/src/main/java/cz/metacentrum/perun/rpc/Api.java

```diff
@@ -255,8 +260,11 @@ else if (isNotEmpty(req.getHeader(OIDC_CLAIM_SUB))) {
 			} else {
 				throw new InternalErrorException("OIDC issuer not send by Authorization Server");
 			}
-			extSourceLoaString = "-1";
-			log.debug("detected OIDC/OAuth2 client for sub={},iss={}",extLogin,iss);
+			var oidcSubIssuer = initOidcAuth(req, additionalInformations);
```

Comment:
on this place we are getting actual sub and issuer  - in the request header(OIDC_CLAIM_SUB, OIDC_CLAIM_ISS)  there are wrong information